### PR TITLE
Virtual table change

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

**Review by randyhoulahan — APPROVED** [review]

File: app/views/meetings/index.js [hunk 1]
```diff
@@ -18,9 +18,12 @@ app.value('ConferenceCal',VueComponent)
 export default ['$location','$scope','$timeout', '$route', '$sce', 'conferenceService', '$q',
         function ($location,$scope,$timeout,  $route, $sce, conferenceService, $q) {
        
+            Vue.component('CbdArticle', CbdArticle)
```
Comment (sbilodeau) [inline app/views/meetings/index.js:21]: are ou sure it's goo idea?

File: app/views/meetings/parallel-meetings.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+import CbdArticle from '~/directives/articles/cbd-article.vue';
+import Vue from 'Vue';
+import _ from 'lodash';
 
 export { default as template } from './parallel-meetings.html'
 
 export default ['$scope', '$route', '$location', 'conferenceService', '$q', '$rootScope',
  function ($scope,  $route, $location, conferenceService, $q, $rootScope) {
        
+            Vue.component('CbdArticle', CbdArticle)
```
Comment (sbilodeau) [inline app/views/meetings/parallel-meetings.js:12]: dbpile resgistration

File: app/views/virtual-tables/index.js [hunk 3]
```diff
@@ -10,12 +10,15 @@ import moment from 'moment';
...
 import '~/services/conference-service'
+import CbdArticle from '~/directives/articles/cbd-article.vue';
+import Vue from 'Vue';
       
 export { default as template  } from './index.html';
 
 export default ['$q', 'user','$http','$scope', '$rootScope', '$timeout', 'articleService', 'angularGridInstance', 'ngDialog', '$route', 'conferenceService',
  function( $q, user,$http, $scope,  $rootScope, $timeout, articleService, angularGridInstance, ngDialog, $route, conferenceService) {
 
+    Vue.component('CbdArticle', CbdArticle)
```
Comment (sbilodeau) [inline app/views/virtual-tables/index.js:21]: tripple registration

**Review by sbilodeau — CHANGES_REQUESTED** [review]
you regsiter same component 3 times globally

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**